Products/DataCollector/plugins/zenoss/cmd/linux/netstat_rn.py

```diff
@@ -110,8 +110,8 @@ def parse(self, results):
             isroute = IP_ROUTE_RECORD_SHORT.search(line)
             if isroute:
                 route.id = isroute.group(1).replace('/', '_')
-                route.mask = isroute.group(1).split('/', 1)[1]
-                if route.mask == 32:
+                route.routemask = isroute.group(1).split('/', 1)[1]
+                if route.routemask == 32:
```

Comment:
Same as comment above: route.routemask is a string and should be treated as such.